crates/matrix-sdk/src/client/builder/homeserver_config.rs

```diff
@@ -30,14 +30,14 @@ use crate::{
 /// Configuration for the homeserver.
 #[derive(Clone, Debug)]
 pub(super) enum HomeserverConfig {
-    /// A precise URL, including the protocol.
+    /// A server name URL, including the protocol.
```

Comment:
Should this be `a homeserver URL`?

Reply:
Indeed! I want to rename all these variants in another PR because it is so confusing.

Reply:
Fixed and rebased.